packages/elastic-apm-synthtrace/src/lib/apm/client/apm_synthtrace_kibana_client.ts

```diff
@@ -58,17 +58,17 @@ export class ApmSynthtraceKibanaClient {
       'https://epr-snapshot.elastic.co/search?package=apm&prerelease=true&all=true&kibana.version=';
     const response = await fetch(url + version, { method: 'GET' });
     const json = await response.json();
-    if (!Array.isArray(json)) {
-      throw new Error('Could not locate apm package compatible with the current kibana version');
-    }
-    const versions = json
-      .map<string>((item) => item.version)
+    const packageVersions = (json ?? []).map((item) => item.version);
+    const validPackageVersions = packageVersions
       .filter((v) => Semver.valid(v))
       .sort(Semver.rcompare);
-    if (versions.length === 0) {
-      throw new Error('Could not locate apm package compatible with the current kibana version');
+    const bestMatch = validPackageVersions[0];
+    if (!bestMatch) {
+      throw new Error(
+        `None of the available APM package versions (${packageVersions}) is a valid match for the current Kibana version (${version})`
```

Comment:
Can you add a way to solve it?

Reply:
E.g a callouts to create an issue?

Reply:
I've updated the copy. 
Who is normally responsible for releasing the APM integration package? Fleet or APM Server?

Reply:
I'm not sure either. Maybe @simitt can clear this out?!

Reply:
APM Server